test/acceptance/cli-test.js

```diff
@@ -1099,6 +1099,182 @@ describe('ember-template-lint executable', function () {
       });
     });
 
+    describe('with sarif formatter', function () {
```

Comment:
Is there a SARIF validator tool? Can we use it here to ensure that our output is valid format?

Reply:
We can validate with ajv, if we want to go that route. The SARIF log has a schema.

Reply:
So for some reason ajv doesn't want to validate using the SARIF schema, as the `$schema` property _within_ the SARIF schema itself is erroring in ajv. Unsure why, but we should for sure look at doing this. Perhaps we can do a follow up?

Reply:
Ya, seems fine to follow up. My hope was mostly just that since this isn't an arbitrary format that we are making up (like existing `--json` output) I'd really like **some** way to make sure we are compliant.

Maybe open a new issue and cross reference?

Reply:
Makes sense.